src/User32/User32+DeviceOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains the <see cref="DeviceOptions"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        public enum DeviceOptions : uint
```

Comment:
This should be named `EnumDisplayDevicesFlags` per our [contribution guidelines](https://github.com/dotnet/pinvoke/blob/master/CONTRIBUTING.md#method-parameter-types)

Reply:
Was borrowing from `MonitorOptions`, but no problem, will change.